### DISCUSSION TITLE
How do you duplicate a file?

### DISCUSSION THREAD

**Discussion by ThSGM** [jarun/nnn#1640] [Q&A] [answered]
Looking at the help, you do ^R for rename/duplicate. If I do so on a file, I have the option of renaming the file. But I don't understand how to duplicate the file or to create a copy.

**Comment by N-R-K** [comment, marked as answer]
To copy a file
* select the file
* go to your destination
* press `p`

To make a copy of a file in the same dir:
* select the file
* press `w` and select "cp as"
* give a new name for the copy

**Comment by jarun** [comment]
The wiki already has the information: https://github.com/jarun/nnn/wiki/Basic-use-cases#duplicate-file

**Comment by 0xACE** [comment]
my favorite is `^R` and then just select the same filename (just hit enter) and it will prompt you again for the duplicated file name.
